src/library_html5.js

```diff
@@ -2653,16 +2653,20 @@ var LibraryJSEvents = {
   emscripten_set_canvas_element_size_main_thread__proxy: 'sync',
   emscripten_set_canvas_element_size_main_thread__sig: 'iiii',
   emscripten_set_canvas_element_size_main_thread__deps: ['emscripten_set_canvas_element_size_calling_thread'],
-  emscripten_set_canvas_element_size_main_thread: function(target, width, height) { return _emscripten_set_canvas_element_size_calling_thread(target, width, height); },
+  emscripten_set_canvas_element_size_main_thread: function(target, width, height) { return _emscripten_set_canvas_element_size_calling_thread(target, width, height);
+},
```

Comment:
This was intentional. I was just trying to split up functions so that multiple functions don't appear on the same line, because that can be kind of misleading. I was trying to avoid things like:
```
function f1(a, b, c) {
  if (/*something*/) /*do stuff*/} function f2 (a, b, c) {
  /*do some unrelated stuff*/
}
```
I had a non-psudocode example a few lines up.

Reply:
Oh, I missed that. So this was working around the intentation in the emitted code? Yeah, it looks wrong sometime. But we should fix that in jsifier.js, probably just need to add or remove an indent someplace. (And in any case, when optimizing this would either be pretty-printed or minified.)

Reply:
> And in any case, when optimizing this would either be pretty-printed or minified.

True, but it does make debugging a bit more annoying.

> But we should fix that in jsifier.js

I can take a quick look at that, but either way I'll revert this line, because it sounds like that's not the correct solution.

Reply:
I reverted the indentation. I also took a look at jsifier.js, but I couldn't really figure it out. I think changing/improving that would be part of another PR anyway.